pkg/trace/gcp.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package trace
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"go.opentelemetry.io/otel/api/trace"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"k8s.io/klog"
+
+	texporter "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/trace"
+	"github.com/pkg/errors"
+	"go.opentelemetry.io/otel/api/global"
+)
+
+const (
+	projectEnvVar  = "MINIKUBE_GCP_PROJECT_ID"
+	parentSpanName = "minikube start"
+)
+
+type gcpTracer struct {
+	projectID string
+	parentCtx context.Context
+	trace.Tracer
+	spans   map[string]trace.Span
+	cleanup func()
+}
+
+func (t *gcpTracer) StartSpan(name string) {
```

Comment:
comment here and or Unexport if not used publicly

Reply:
added comment